cratedb_toolkit/docs/settings.py

```
@@ -0,0 +1,773 @@
+
+def extract_cratedb_settings() -> Dict[str, Dict[str, Any]]:
+    """
+    Extract CrateDB settings from the documentation website.
+
+    Returns:
+        Dict[str, Dict[str, Any]]: Dictionary of settings with their properties
+    """
+    settings = {}
+
+    with console.status("[bold green]Fetching documentation...", spinner="dots"):
+        response = requests.get(DOCS_URL)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)